### ISSUE TITLE
[QUESTION]how to install on fedora 37?

### ISSUE THREAD

**Issue by radical4ever** [Frogging-Family/glibc-eac#4]
hi, i would love to know if this glibc is possible to install for fedora 37. thank you

**Comment by Tk-Glitch** [comment]
Not possible as is, it's made for pacman distros such as Arch Linux. However, you can use the patches (https://github.com/Frogging-Family/glibc-eac/blob/main/e47de5cb.patch and https://github.com/Frogging-Family/glibc-eac/blob/main/rogue_company_reverts.patch) on libc sources and maybe make a Fedora package with the result?
Alternatively, GE offers a Fedora version of this https://copr.fedorainfracloud.org/coprs/gloriouseggroll/nobara/package/glibc/

**Comment by radical4ever** [comment]
> Not possible as is, it's made for pacman distros such as Arch Linux. However, you can use the patches (https://github.com/Frogging-Family/glibc-eac/blob/main/e47de5cb.patch and https://github.com/Frogging-Family/glibc-eac/blob/main/rogue_company_reverts.patch) on libc sources and maybe make a Fedora package with the result? Alternatively, GE offers a Fedora version of this https://copr.fedorainfracloud.org/coprs/gloriouseggroll/nobara/package/glibc/

And how do i use the patches?(im using fedora 37)

**Comment by begin-theadventure** [comment]
Maybe try installing GE's glibc version with rpm package: https://copr.fedorainfracloud.org/coprs/gloriouseggroll/nobara/build/4770029/

**Comment by radical4ever** [comment]
> Maybe try installing GE's glibc version with rpm package: https://copr.fedorainfracloud.org/coprs/gloriouseggroll/nobara/build/4770029/

Yeah i see there is already a package but the problem is that is for an older version of what i have (my version is 2.36 for fedora 37 gnome43), and i thinking if im risking my pc downgrading the version of glibc

**Comment by begin-theadventure** [comment]
Timeshift and try to install it. / Switch to Nobara. / Stick with Fedora's glibc.
Three ways from me.

**Comment by begin-theadventure** [comment]
Timeshift and try to install it. / Switch to Nobara. / Stick with Fedora's glibc.
